lua/autorun/client/cl_downloader_init.lua

```diff
@@ -0,0 +1,544 @@
+                if IsValid(self.frame) then
+                    self:SaveCache()
+                end
+            end)
+        end
+    end
+end
+
+-- Filter the addons by type, always sorted by name
+function Browser:FilterAddons(selectedValue)
+    -- Getting addons
+    local addons = engine.GetAddons()
+
+    table.sort(addons, function(a, b) return string.lower(a.title) < string.lower(b.title) end)
+
+    -- Move all icons to the void and hide them
+    for k, addonInfo in ipairs(addons) do
+        local wsid = tonumber(addonInfo.wsid)
+        local scanInfo = self.scanResult[wsid]
+
+        if not scanInfo then continue end
+
+        if IsValid(scanInfo.iconArea) then
+            scanInfo.iconArea:SetParent(self.addonsList.void)
+            scanInfo.iconArea:Hide()
+        end    
+    end
+
+    -- Move the selected icons back to the panel and make sure they appear
+    for k, addonInfo in ipairs(addons) do
+        local wsid = tonumber(addonInfo.wsid)
+        local scanInfo = self.scanResult[wsid]
+
+        if not scanInfo then continue end
+
+        if IsValid(scanInfo.iconArea) then
+            if selectedValue == FILTER.ALL then
+                scanInfo.iconArea:Show()
+                scanInfo.iconArea:SetParent(self.addonsList)
+            elseif selectedValue == FILTER.SELECTED then
+                if scanInfo.manual == nil and scanInfo.cachedManual == nil and scanInfo.selected == true or
+                   scanInfo.manual == nil and scanInfo.cachedManual == true or
+                   scanInfo.manual == true
+                then
+                    scanInfo.iconArea:Show()
+                    scanInfo.iconArea:SetParent(self.addonsList)
+                end
+            elseif selectedValue == FILTER.IGNORED then
+                if scanInfo.manual == nil and scanInfo.cachedManual == nil and scanInfo.selected == false or
+                   scanInfo.manual == nil and scanInfo.cachedManual == false or
+                   scanInfo.manual == false
+                then
+                    scanInfo.iconArea:Show()
+                    scanInfo.iconArea:SetParent(self.addonsList)
+                end
+            elseif selectedValue == FILTER.MANUAL then
+                if scanInfo.manual == nil and scanInfo.cachedManual ~= nil and scanInfo.cachedManual ~= scanInfo.selected or
+                   scanInfo.manual ~= nil and scanInfo.manual ~= scanInfo.selected
+                then
+                    scanInfo.iconArea:Show()
+                    scanInfo.iconArea:SetParent(self.addonsList)
+                end
+            end
+        end  
+    end
+
+    -- Refresh the scroll bar
+    timer.Simple(0.2, function()
+        if IsValid(self.frame) and IsValid(self.addonsScroll) then
+            self.addonsScroll:InvalidateLayout()
+        end
+    end)
+end
+
+concommand.Add("uwd_page", function()
+    steamworks.ViewFile(2214712098)
+end)
+
+concommand.Add("uwd_menu", function()
```

Comment:
Shouldn't we maintain the pattern of prefixing the commands with `downloader_`?

Reply:
Yep. Now I'm using the correct command prefixes.